sci-libs/calculix-cgx/calculix-cgx-2.8.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit eutils
+
+MY_P=cgx_${PV}
+
+DESCRIPTION="A Free Software Three-Dimensional Structural Finite Element Program"
+HOMEPAGE="http://www.calculix.de/"
+SRC_URI="http://www.dhondt.de/${MY_P}.all.tar.bz2
+	doc? ( http://www.dhondt.de/${MY_P}.pdf )"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+# nptl removed since I cannot work around it
+IUSE="doc examples"
+
+RDEPEND="media-libs/mesa[nptl]
+	>=media-libs/freeglut-1.0"
+DEPEND="${RDEPEND}
+	doc? ( app-text/ghostscript-gpl )"
+
+S=${WORKDIR}/CalculiX/${MY_P}/src/
+
+src_unpack() {
+	default
+	cp "${DISTDIR}/${MY_P}.pdf" "${S}"
```

Comment:
I think, you only want to do that if `doc` is enabled and missing die. Better:
`use !doc || cp "${DISTDIR}/${MY_P}.pdf" "${S}" || die`